assets/mail-templates/success.txt

```diff
@@ -0,0 +1,14 @@
+✔️ Echidna just published “@TITLE”
+  (@SHORTNAME), a @TYPE:
+
+  👍 @URL
+
+More details are available here@ATTACHMENT:
+
+  https://labs.w3.org/echidna/api/status?id=@ID
```

Comment:
I wonder if we should use `global.BASE_URI` or `process.env.BASE_URI` here, instead of hard-coding the URL?